sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -807,16 +805,15 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the updated device twin object",
+            "description": "Returns the updated device twin object.",
             "schema": {
               "$ref": "#/definitions/Twin"
             }
           }
         }
       },
       "patch": {
-        "summary": "Updates tags and desired properties of a device twin.",
-        "description": "Updates a device twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Updates the tags and desired properties of a device. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
```

Comment:
Same "Updates a device twin" seems better.

Reply:
For PATCH, just update seems correct

Reply:
We need to call out that all you can update are tags and desired properties; this is the service-side contract. 
Can you clarify what your second statement means?

Reply:
Second statement means nothing. I was not aware PUT for twins does not create. I was just pointing out that description for PATCH seems correct but Create was missing in PUT. You cleared that.

Reply:
Ah, got it!